x/shared/types/params.go

```diff
@@ -65,3 +74,14 @@ func ValidateNumBlocksPerSession(v interface{}) error {
 
 	return nil
 }
+
+// ValidateClaimWindowOpenOffsetBlocks validates the ClaimWindowOpenOffsetBlocks param
+// NB: The argument is an interface type to satisfy the ParamSetPair function signature.
+func ValidateClaimWindowOpenOffsetBlocks(v interface{}) error {
```

Comment:
Let's ensure it's greater than zero and add a test.

Reply:
I could imagine adding a test for the error case, where the input is not a uint64 but I don't think it's totally necessary. `ValidateClaimWindowOpenOffsetBlocks()` is only called in two places:

![image](https://github.com/pokt-network/poktroll/assets/600733/b2b67b05-1614-4cb5-b0f8-634de3893cc4)
> Fig. 1. Usages.

![image](https://github.com/pokt-network/poktroll/assets/600733/f1344b24-aab2-4a42-9167-ef2183f34612)
> Fig 2. Usage in `shared.msgServer#UpdateParam()`, where it is guaranteed to be a uint64.

![image](https://github.com/pokt-network/poktroll/assets/600733/a7a87740-1581-47a9-bb49-cf7e58debefd)
> Fig 3. Usage in `sharedtypes.Params#ParamSetPairs()`, which is called by the cosmos-sdk framework and seems to be related to migrations somehow.

Reply:
Whether it's an explicit check or a unit test, either works.